templates/apps/create_base.html

```diff
@@ -151,36 +159,30 @@ <h1 class="h3 mb-3 card-title">Create {{ form.model_name }}</h1>
 
         // On keydown, clear the countdown
         $("#id_subdomain").on('keydown', function () {
-            clearTimeout(typingTimer);
+                clearTimeout(typingTimer);
```

Comment:
This shouldn't really matter, but still, why is there this indent?

Reply:
This wasn't a deliberate change. Will fix this.